apps/client/src/translation/languages/lv.ts

```diff
@@ -0,0 +1,33 @@
+export const langLV = {
+  'common.expected_finish': 'Paredzamais beigu laiks',
+  'common.minutes': 'min',
+  'common.now': 'Tagad',
+  'common.next': 'Pēc tam',
+  'common.public_message': 'Publiskā ziņa',
+  'common.scheduled_start': 'Plānotais sākums',
+  'common.scheduled_end': 'Plānotās beigas',
+  'common.projected_start': 'Paredzamais sākums',
+  'common.projected_end': 'Paredzamās beigas',
+  'common.stage_timer': 'Skatuves taimeris',
+  'common.started_at': 'Sākās',
+  'common.time_now': 'Laiks tagad',
+  'common.no_data': 'Nav datu',
+  'countdown.ended': 'Plāna punkts beidzās',
```

Comment:
Are you sure this is correct?
Is `punkts` a point? is there a better word for event?